### ISSUE TITLE
Redirect links not working

### ISSUE THREAD

**Issue by kayjan** [AccentDesign/karma_sphinx_theme#4]
Items in autosummary table has the correct # redirection, but when pressed it does not get redirected.

**Comment by stuartaccent** [comment]
hi @kayjan,

just to let you know I am aware of your issues / questions. Just to let you know this package was thrown together 5 years ago mainly to support a css package we wrote to see if it covered the basics. The theme was taken from another (though can't remember which now) and the css just replaced.

I will get the package upto date and building again and we can take it from there. I will try to get this done this week.

Are you referring to the [sphinx.ext.autosummary](https://www.sphinx-doc.org/en/master/usage/extensions/autosummary.html#module-sphinx.ext.autosummary) extension?

Thanks
Stu

**Comment by kayjan** [comment]
Yes I'm referring to that extension that is not working with the theme :)
